### PR TITLE
Allow apache to bind to port 5000 under SELinux

### PR DESCRIPTION
**Pull request by rgarrigue**
Allow apache httpd binding to 5000/tcp

Solve #200

### CODE REVIEW

File: manifests/apache.pp [hunk 1]
```diff
@@ -123,4 +123,6 @@
     group   => 'root',
     mode    => '0644',
   }
+  
+  selinux::boolean { 'httpd_use_openstack': }
```
Comment (ekohl) [inline manifests/apache.pp]: We're not using a module for selinux. That means we could use ```selboolean``` which is built in, but then you need to be sure that selinux is enabled (either enforcing or permissive).

It's also only needed for crane so it belongs in ```pulp::crane::apache```.

**Comment by ekohl** [comment]
@rgarrigue could you have a look at my comment?

**Comment by rgarrigue** [comment]
@ekohl saw that. Will see if I can fix this PR this week.

**Comment by ekohl** [comment]
Btw, I think the reason we're not hitting this in the Katello scenario is that katello-selinux takes care of this.

**Comment by sean797** [comment]
Should this be done in https://github.com/pulp/pulp/blob/master/server/selinux/server/pulp-server.te instead of katello-selinux?

**Comment by ekohl** [comment]
I do think that would make sense but not an expert on selinux. Going to close this since @rgarrigue removed his fork so I think it's unlikely he's going to work on this.

**Pull request closed without merging**